src/java/org/apache/cassandra/tools/nodetool/stats/TableStatsHolder.java

```diff
@@ -368,6 +374,14 @@ private String format(long bytes, boolean humanReadable)
         return humanReadable ? FileUtils.stringifyFileSize(bytes) : Long.toString(bytes);
     }
 
+    private Map<String, String> format(Map<String, Long> map, boolean humanReadable)
```

Comment:
nit: it formats the size. Maybe call it `formatSize` to provide better clarity as the method is only called on some values.

Reply:
`org.apache.cassandra.tools.nodetool.stats.TableStatsHolder#format(long, boolean)` shares the same name and does the same thing, for this reason I rather leave it as `format` here as it extends the `long` type to handle a map with `long` values